pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -286,7 +310,7 @@ var _ = Describe("ImageUpload", func() {
 		})
 
 		It("PVC does not exist --pcvc-size", func() {
-			testInit(http.StatusOK)
+			testInit(http.StatusOK, true)
```

Comment:
if we still support the non-async func, maybe we should dedicate a context for it? or run a table test (one time with the async func and one time w/o)

Reply:
Sure I will make it a table. We are testing it goes through the steps of falling back to the sync endpoint.

Reply:
:+1:

Reply:
do we still miss the explicit test for the synchronous flow or is this resolved?

Reply:
Right above here we have the table where we go through both options. The rest of the flow doesn't care which option we picked, we just need to exercise both options at least once to make sure they work.

Reply:
Reworked it a little, so most code didn't change and now we have 2 inits one with async for the test that actually uses it, and one that defaults the value, so we don't have to change calls everywhere.

Reply:
Nice :+1: thank you